container/nginx/conf.d/shared.conf

```diff
@@ -62,7 +62,7 @@ location / {
     # $arg_filename, $arg_download, $param_go_get, $http_if_none_match
     proxy_cache_key             $uri$format_final$arg_depth$arg_bytes$arg_filename$arg_download$param_go_get$http_if_none_match;
     proxy_cache_lock            on;
-    proxy_cache_valid           200 301 302 410     365d;
```

Comment:
Shouldn't we cache 403s then?

Reply:
I supposed we could, but won't that interfere with other 403s on the shim codebase, such as [here](https://github.com/filecoin-saturn/L1-node/blob/main/container/shim/src/modules/registration.js#L234)? - I think this is the only place.

Reply:
Has it's own proxying config https://github.com/filecoin-saturn/L1-node/blob/51582eeba5b5453f1900978339ccf5289590c524/container/nginx/conf.d/register.conf

Reply:
addressed